03_launch_mgmt_cluster.sh

```diff
@@ -234,13 +234,18 @@ EOF
 # Create the BMH CRs
 #
 function make_bm_hosts() {
+  if [ "${LIBVIRT_FIRMWARE}" = "bios" ]; then
+    BOOT_MODE="legacy"
+  else
+    BOOT_MODE="UEFI"
+  fi
```

Comment:
@fmuyassarov good point, we'd need to also set `libvirt_secure_boot` ref https://github.com/metal3-io/metal3-dev-env/commit/1241e2954026068fd2e88232178e64e0b8c0c317

But perhaps it would be a more intuitive user experience if we just exposed `BOOT_MODE` so that the value aligns with the BMH `bootMode` API, then automatically select the required firmware based on that setting.

We might also want to consider making UEFI the default so that it aligns with the BMH defaults?